src/OpenSSL/SSL.py

```diff
@@ -701,6 +711,35 @@ def set_default_verify_paths(self):
         """
         set_result = _lib.SSL_CTX_set_default_verify_paths(self._context)
         _openssl_assert(set_result == 1)
+        num = self._check_num_store_objects()
```

Comment:
``SSL_CTX`` uses lazy loading for ``capath`` directory. The check will always return 0 when a system has only ``capath`` and not a ``cafile``.